### ISSUE TITLE
Add Data for Maryland

### ISSUE THREAD

**Issue by JesseCrocker** [OpenBounds/OpenHuntingData#78]

**Comment by kdough03** [comment]
State only has point features: http://data.imap.maryland.gov/datasets/74a7ef2281bc4bb6b3105f7de2a96bd4_31

Will contact directly.

**Comment by kdough03** [comment]
11/2 message reply from DNR:

"I am not aware of a file that contains all hunting zones managed by DNR.  As Matt indicated, the DNR Lands and Conservation Easements shapefile in the Protected Lands category at http://dnrweb.dnr.state.md.us/gis/data/ will provide the areas for most of those listed in the regulations.  I did a quick check and a couple areas I noticed not included are the Liberty and Loch Raven watershed areas in Baltimore County that are owned by Baltimore City.  I am sure there are other examples.

I did see the following map of Chesapeake Forest public hunting tracts:

http://dnr2.maryland.gov/forests/Pages/chesapeakeforestlands_publichuntingmaps.aspx

I am checking with staff in the Forest group and Wildlife group in DNR for more details."
